charts/posthog/values.yaml

```diff
@@ -1534,13 +1534,9 @@ loki:
   # have an anti-affinity so you'll need at least as many nodes as replicas in a
   # set.
   enabled: false
+
   loki:
-    # For backwards compatibility with Loki Chart 2.x we disable multi-tenancy.
```

Comment:
I found this comment a bit misleading, we disable authentication, not multi-tenancy